src/terminal/parser/telemetry.hpp

```diff
@@ -85,6 +85,8 @@ namespace Microsoft::Console::VirtualTerminal
             OSCFG,
             OSCBG,
             DECALN,
+            XTPUSHSGR,
+            XTPOPSGR,

```

Comment:
I believe you also need to add logging entries for these codes in the telemetry.cpp file similar to this:
https://github.com/microsoft/terminal/blob/84b1b874dd09dd11c5099169acfcbb28c7ce0609/src/terminal/parser/telemetry.cpp#L249

Reply:
Thanks!